examples/api/test/material/dropdown_menu/dropdown_menu.1_test.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/dropdown_menu/dropdown_menu.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('DropdownMenu', (WidgetTester tester) async {
```

Comment:
Consider using a more meaningful test description.

Reply:
What do you think of [test: Add better test description](https://github.com/flutter/flutter/pull/149547/commits/47327c5a9e60036036a420c94990547d2f5b9038) ?